src/implementations/minecraft/mod.rs

```diff
@@ -351,6 +358,55 @@ impl MinecraftInstance {
         )
         .await?;
 
+        // Step 3 (part 2): Forge Setup
+        if let Flavour::Forge { build_version } = flavour.clone() {
+            let _ = event_broadcaster.send(Event {
+                event_inner: EventInner::ProgressionEvent(ProgressionEvent {
+                    event_id: progression_event_id,
+                    progression_event_inner: ProgressionEventInner::ProgressionUpdate {
+                        progress: 1.0,
+                        progress_message: "3/4: Installing Forge Server".to_string(),
+                    },
+                }),
+                details: "".to_string(),
+                snowflake: Snowflake::default(),
+                caused_by: CausedBy::Unknown,
+            });
+
+            let jre = path_to_runtimes
+                .join("java")
+                .join(format!("jre{}", jre_major_version))
+                .join(if std::env::consts::OS == "macos" {
+                    "Contents/Home/bin"
+                } else {
+                    "bin"
+                })
+                .join("java");
+
+            if !dont_spawn_terminal(
+                Command::new(&jre)
+                    .arg("-jar")
+                    .arg(&config.path.join("forge-installer.jar"))
+                    .arg("--installServer")
+                    .arg(&config.path)
+            )
+            .stdout(Stdio::null())
+            .stdin(Stdio::null())
+            .stderr(Stdio::null())
+            .spawn().map_err(|_| eyre!("Failed to start forge-installer.jar"))?
+            .wait().await.map_err(|_| eyre!("forge-installer.jar failed"))?
```

Comment:
Use `.context()` instead. This way you get both the original error and a context message